src/core/gba/gbaLink.cpp

```diff
@@ -41,6 +41,9 @@
 #define snprintf _snprintf
 #endif
 
+#ifndef UPDATE_REG
+#undef UPDATE_REG
+#endif
```

Comment:
I don't understand this block, it undefines `UPDATE_REG` if it is already undefined. Leftover from an earlier patch?